docs/guides/misc/setting-static-ip.md

```diff
@@ -0,0 +1,78 @@
+[DRAFT]
+
+When you plug your device into the network, you will be assigned an IP address from the router. For most use cases, this is all that is needed, but for servers it is strongly recommended that the IP address is made static.
+
+This means ensuring that no matter how many times the device is rebooted / disconnected and reconnected it will _always_ get the same IP address.
+
+Depending on your operating system / device. There are many ways to achieve this, we will list a few here.
+
+# Raspbian / Raspberry Pi OS
+
+First, lets make take some notes about our current network settings with the following command:
+
+```bash
+ip r | grep default
```

Comment:
The following allows to print the default route directly, so no need to filter with `grep`:
```sh
ip r l 0/0
```